cloudslang-entities/src/main/java/io/cloudslang/lang/entities/ListForLoopStatement.java

```diff
@@ -20,7 +20,7 @@
  *
  * @author Bonczidai Levente
  */
-public class ListForLoopStatement extends ForLoopStatement implements Serializable{
+public class ListForLoopStatement extends ForLoopStatement implements Serializable {
```

Comment:
yes.. they both contain a string (varName) .. but we need to make **clear distinction** between the cases.

We could add a superclass that has this field, but then we would have either empty "end" classes or that superclass would have a field `async` - if I remember correctly that was the initial approach but we agreed that is not a good approach and we prefer subclassing.